tests/test_Uxxx.py

```diff
@@ -1,13 +1,10 @@
 import inspect
 
 import pytest
-from rlp import decode

```

Comment:
This is a result of `reorder_python_imports` which is used by flask project. I brought most of the formatting rules from this project. https://github.com/asottile/reorder_python_imports https://github.com/pallets/flask/blob/master/.pre-commit-config.yaml